index.js

```diff
@@ -0,0 +1,134 @@
+const fs = require('fs')
+const path = require('path')
+const fetch = require('node-fetch')
+const colors = require('colors/safe')
+
+
+//Función que desencadena todo el proceso (valida si el archivo tiene terminación .md)
+function validateFile(file,options){
+    if(path.extname(file) === '.md'){
+        readFile(file,options)
+    }else{
+        console.log(colors.magenta.bold('*****Introduce un archivo con una extensión válida(.md)*****'))
+    }
+}
+
+//Leyendo archivo, generando string
+function readFile(file,options){
+    const string = fs.readFileSync(file, 'utf-8')
+    getLinks(string,options)
```

Comment:
en algunos lenguajes string es una palabra reservada y puede generar error.
```suggestion
    getLinks(arryaLinks,options)
```